### PR TITLE
Moved support to homepage

### PR DESCRIPTION
**Pull request by MichaelRupertDev**
https://github.com/borgbackup/borg/issues/2812

I still need to add the actual commercial support providers, need to get that list.

### CODE REVIEW

File: support.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+If you want to encourage developers to fix some specific issue or implement some
+specific feature suggestion, you can post a new bounty or back an existing one
+(they always refer to an issue in our `issue tracker`_).
+
+As a developer, you can become a Bounty Hunter and win bounties (earn money) by
+contributing to Borg, a free and open source software project.
+
+We might also use BountySource to fund raise for some bigger goals.
+
+.. _security-contact:
```
Comment (ThomasWaldmann) [inline support.rst]: this stuff is linked from elsewhere in the docs, so we need a solution for that.

File: support.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+Security
+--------
+
+In case you discover a security issue, please use this contact for reporting it privately
+and please, if possible, use encrypted E-Mail:
+
+Thomas Waldmann <tw@waldmann-edv.de>
+
+GPG Key Fingerprint: 6D5B EF9A DD20 7580 5747  B70F 9F88 FB52 FAF7 B393
```
Comment (ThomasWaldmann) [inline support.rst]: the FP is also needed to verify the borg download.

File: support.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+Twitter
+-------
+
+Follow @borgbackup for announcements. You can also add @borgbackup if you
+would like to get retweeted for a borg related tweet.
+
+Please understand that Twitter is not suitable for longer / more complex
+discussions - use one of the other channels for that.
+
+Bounties and Fundraisers
```
Comment (ThomasWaldmann) [inline support.rst]: the "fund development" link from index.html points here (in the docs).

File: support.rst (hunk 2 above)
Comment (MichaelRupertDev) [reply support.rst]: Should we keep the security info around in the docs as a "Security" section and move the rest of support over?

Comment (ThomasWaldmann) [reply support.rst]: yes, guess that would work.

File: index.rst [hunk 4]
```diff
@@ -57,6 +57,26 @@ BorgBackup – Deduplicating archiver with compression and authenticated encrypt
...
         </div>
+
+    </div>
+
+    <br>
+
+    <div id="links">
+        <div class="link" style="width: 400px;">
+            <a href="/support/support.html">
+                <span class="action">Recieve</span><br>
```
Comment (ThomasWaldmann) [inline index.rst]: Receive

**Comment by MichaelRupertDev** [comment]
I've updated the support section. This is the current homepage: 
screen shot 2017-11-24 at 7 13 36 pm

File: support/commercial.rst [hunk 5]
```diff
@@ -0,0 +1,12 @@
+.. class:: hide-rst-heading
+
+Paid Support Services
```
Comment (ThomasWaldmann) [inline support/commercial.rst]: Maybe rather "Paid Support and Services"

e.g. rsync.net could be also here, it is service-only (backup storage provider), no support.

File: support/support.rst [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+As usual on IRC, just ask or tell directly and then patiently wait for replies.
+Stay connected.
+
+You could use the following link (after connecting, you can change the random
+nickname you get by typing "/nick mydesirednickname"):
+
+http://webchat.freenode.net/?randomnick=1&channels=%23borgbackup&uio=MTY9dHJ1ZSY5PXRydWUa8
+
+.. _mailing_list:
```
Comment (ThomasWaldmann) [inline support/support.rst]: this is likely also referenced from the docs.

Comment (MichaelRupertDev) [reply support/support.rst]: I've addressed this in the borg PR.

File: support/commercial.rst [hunk 7]
```diff
@@ -0,0 +1,12 @@
+.. class:: hide-rst-heading
+
+Paid Support and Services
+=========================
+
+Here are some commercial support providers for borgbackup:
```
Comment (ThomasWaldmann) [inline support/commercial.rst]: "service and support providers"

File: Makefile [hunk 8]
```diff
@@ -2,6 +2,8 @@
 PAGES=\
 index.html\
 demo.html\
+support/support.html\
+support/commercial.html\
```
Comment (ThomasWaldmann) [inline Makefile]: nitpick: support/free.html and support/commercial.html would be more consistent.

**Comment by ThomasWaldmann** [comment]
OK, one minor nitpick left, but otherwise it's finished, right?

**Comment by MichaelRupertDev** [comment]
Updated, should be good to go.

**Comment by ThomasWaldmann** [comment]
ok, thanks for the PR!

**Pull request merged**